src/Network/Network.Test/ScenarioTests/ApplicationGatewayTests.ps1

```diff
@@ -4171,6 +4171,69 @@ function Test-ApplicationGatewayFirewallPolicyManagedRuleGroupOverrideEmptyRule
 	}
 }
 
+<#
+.SYNOPSIS
+Application gateway v2 top level waf tests in ManagedRuleGroupOverride with sensitivity
+#>
+function Test-ApplicationGatewayFirewallPolicyManagedRuleGroupOverrideWithSensitivity
```

Comment:
Please add another test checking for configuring sensitivity override on another ruleset (Bot/CRS) - it should return error

Reply:
actually I did test that and it returned error, I had it in the group chat, let me see if I can write it though

Reply:
tested manually